auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -14,23 +14,34 @@ class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
     @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
+    def _auth_timeout_get_parameter_delay(self, db):
```

Comment:
I 100% agree with your design choice. The solution here is to basically just create a `get_session_parameters` method that returns the output of the two new methods in a compatible tuple